sdcm/mgmt/operator.py

```diff
@@ -420,3 +428,8 @@ def add_cluster(self, name, host=None, db_cluster=None, client_encrypt=None, dis
 
     def upgrade(self, scylla_mgmt_upgrade_to_repo):
         raise NotImplementedError()
+
+    def feature(self, command: str, feature: str) -> bool:
+        if command == "repair" and feature == "single-host-parallelism":
```

Comment:
can you explain this logic please, why would operator not support it ?

Reply:
I rather have here `raise NotImplementedError()` until someone actually test it, and verify this works or not on operator,

after it's already established we aren't gonna use this on nemesis for now.